tests/Admin/AdminTest.php

```diff
@@ -745,8 +745,6 @@ public function testGetLabelTranslatorStrategy(): void
     {
         $admin = new PostAdmin('sonata.post.admin.post', 'NewsBundle\Entity\Post', 'Sonata\NewsBundle\Controller\PostAdminController');
 
-        $this->assertNull($admin->getLabelTranslatorStrategy());
```

Comment:
I assume you remove this because it throws a TypeError.

IMHO, We should throw a LogicException in these getter when the value is null.
And we could then add a test with `$this->expectException(\LogicException::class)`.

But I'd like the @sonata-project/contributors point of view.
WDYT @jordisala1991 ?

Reply:
I would be a good idea, yes. I removed it to just make the test pass, but would be better to add a test instead

Reply:
I tried to do something, but not sure if I get the idea correctly. Also, LogicException messages could probably be more meaningful, I wasn't sure what to put so to be "generic" for now.

Reply:
You got the idea correctly. I juste made a minor comment on the exception message for consistency with others LogicException until we got something better.